### PR TITLE
rename interest rate to "WACC"

### LINKED ISSUE #223: rename interest rate to "WACC"

### PR DESCRIPTION
**Pull request by joAschauer**
Fixes #223

### CODE REVIEW

File: ptxboa/static/__init__.py [hunk 1]
```diff
@@ -1,807 +1,807 @@
...
-
-
-TransportType = Literal["Ship", "Pipeline"]
-TransportValues = ["Ship", "Pipeline"]
-
-
-OutputUnitType = Literal["USD/MWh", "USD/t"]
-OutputUnitValues = ["USD/MWh", "USD/t"]
+# -*- coding: utf-8 -*-
+"""DO NOT EDIT (created by _update_static.py)."""
```
Comment (markushal) [inline ptxboa/static/__init__.py:2]: Did you just replace "interest rate" with "WACC" in this file, or did you change it in the database as well?

Comment (joAschauer) [reply ptxboa/static/__init__.py:2]: it is updated in the database as well as in the excel file which feeds to the database

Comment (markushal) [reply ptxboa/static/__init__.py:2]: nice!

**Comment by wingechr** [comment]
@markushal I don't have this error

**Comment by joAschauer** [comment]
> choosing "WACC" in the input data tab raises an exception:

also cannot reproduce

**Comment by wingechr** [comment]
I think we should merge it and then check locally with @markushal

**Comment by markushal** [comment]
Hmmm. That is weird. I just tried again, and I also cant reproduce the error. Then lets merge it.

**Review by markushal — APPROVED** [review]

**Pull request merged**